src/app/App.scss

```diff
@@ -37,6 +38,15 @@ $header-text-color: #fff;
     display: inline-block;
   }
 
+  &__container-remove-button {
+    display: inline-block;
+  }
+
+  &__remove-button {
+    margin-left: 15px !important;
```

Comment:
is it necessary to add important? If the answer is yes, can we add an override scss file for this purpose?

Reply:
Yeah, if I don't put the !important, It doesn't take it